Gulpfile.mjs

```diff
@@ -424,6 +424,7 @@ function copyDts(packages) {
 const libBundles = [
   "packages/babel-parser",
   "packages/babel-plugin-proposal-optional-chaining",
+  "packages/babel-preset-react",
```

Comment:
everything is getting bundled huh 😄, feel like eventually this list will be the opposite?

Reply:
In Babel 8 I'd like to eventually bundle everything, so that we don't pay the cost of resolving all the files (some packages have many of them)